pom.xml

```diff
@@ -138,6 +145,30 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
```

Comment:
Can you explain what those two profiles are for ?

Reply:
noIbex profile is used to launch tests of the groups 1s, 10s and checker, avoiding build to fail locally when Ibex is not installed

dev profile is here to have a default profile as teh two others are declared not active by default.

Reply:
Ok. How to we use the profiles?

Reply:
In command line, mvn has a parameter to select the profile: -P. So let say you want to locally build (and install in your local maven repository) Choco Solver using the noIbex profile because you don't have Ibex installed on your computer. You can do it with the following line:
`mvn clean install -P noIbex` or `mvn clean install -PnoIbex`